bin/uno

```diff
@@ -33,6 +33,10 @@ case "$1" in
     check_dirs ACCUMULO_HOME
     "$ACCUMULO_HOME"/bin/accumulo shell -u "$ACCUMULO_USER" -p "$ACCUMULO_PASSWORD" "${@:2}"
     ;;
+  zk)
+    check_dirs ZOOKEEPER_HOME 
+    "$ZOOKEEPER_HOME"/bin/zkCli.sh
```

Comment:
Should we chroot this to Accumulo's dir? (Might not be useful for tracer server, since that still uses stuff outside that path, but could be convenient for most scenarios.)